specification/search/data-plane/Azure.Search/readme.md

```diff
@@ -171,6 +171,58 @@ directive:
       - RequiredReadOnlyProperties
 ```
 
+### Suppress custom user schema model violations
+Many Search operations return a custom user schema for the documents in an index.  Some of them are incorrectly specified as `"additionalProperties": { "type": "object" }` instead of `"additionalProperties": true`.  We’ve filed https://github.com/Azure/azure-rest-api-specs/issues/23807 to track fixing these, but are temporarily suppressing the model violation warnings to avoid impacting the next release our generated client libraries.
+
+
+``` yaml
+directive:
+  - suppress: INVALID_TYPE
+    from: searchservice.json
+    reason: Temporarily ignore warnings about incorrectly specified custom user schemas.
+  - suppress: INVALID_TYPE
+    from: searchindex.json
+    reason: Temporarily ignore warnings about incorrectly specified custom user schemas.
+```
+
```

Comment:
Let's remove this for now.  We've got the tracking bug filed at https://github.com/Azure/azure-rest-api-specs/issues/23807 which I think is enough to move forward.
```suggestion
```